examples/neural_networks/plot_elm_hyperparameters.py

```diff
@@ -0,0 +1,80 @@
+"""
+===========================================================
+Extreme Learning Machines: Effect of tuning hyperparameters
+===========================================================
+
+This example first demonstrates how sklearn.grid_search.GridSearchCV can
+scan through the space of 2 significant hyperparameters C and weight_scale,
+recording their cross-validation scores on the digits dataset.
+Please note that n_hidden is also a significant parameter but was not
+demonstrated here because of the involved computational costs.
```

Comment:
I would argue that his example runs too long to be a "plot" example (20 minutes).